dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -38,6 +39,61 @@
 		};
 	};
 
+	clocks {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		clk_hse: clk-hse {
+			#clock-cells = <0>;
+			compatible = "st,stm32-hse-clock";
+			/* H7 clock driver may not always need this value */
+			/* but it is required by the binding */
+			clock-frequency = <DT_FREQ_M(25)>;
```

Comment:
why setting 25MHz then ?  Even if not used here, most of the stm32h7  have `clock-frequency = <DT_FREQ_M(8)>;``

Reply:
25MHz is value of HSE_VALUE provided by Cube HAL